Prace_domowe/Praca_domowa1/Grupa4/ulasik_kinga/PD1_forest_fires_EDA.ipynb

```diff
@@ -0,0 +1,1825 @@
+{
```

Comment:
<p>Gratuluję podejścia, nikt nie liczył średniej, więc pół godziny zastanawiałem się czemu u Ciebie wygląda to inaczej.</p><p> </p><p>Czy usuwanie outlierów ma sens? Wtedy tracimy informację czy tam las się palił czy nie. Może warto je zczapkować?</p><p> </p><p>Jeśli jest mała średnia to może być tak, że las pali się w każdy piątek, ale na małej powierzchni albo raz, ale porządnie</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Reply:
<p>Gratuluję raz jeszcze, bo średnia jest lepsza od sumy, ale warto też dodać odchylenie, by uwzględnić ostatnią uwagę</p>